proto/package.json

```diff
@@ -4,7 +4,7 @@
     "@improbable-eng/grpc-web": "^0.14.0",
     "apache-arrow": "5.0.0",
     "flatbuffers": "^1.12.0",
```

Comment:
flatbuffers too please, and apache-arrow... that said i'm unsure if we actually use apache arrow at all here.

Reply:
Done, and regenerated package-lock.json.